scripts/perf_report.py

```diff
@@ -9,10 +9,42 @@
 """
 import sys
 import argparse
+import statistics
 from collections import OrderedDict
 from collections import namedtuple
 from prettytable import PrettyTable
 
+# ##########################################################
+# Layout of a summary row from the output file:
+# ##########################################################
+# pylint: disable-next=line-too-long
+# Summary: For 32 clients, Baseline - No logging, num_threads=1, num_ops=32000000 (32 Million) ops, Elapsed time=105613818950 (~105.61 Billion) ns, Avg. Elapsed real time=3300 ns/msg, Server throughput=302990 (~302.99 K) ops/sec, Client throughput=10035 (~10.03 K) ops/sec, Avg. ops per thread=32000000 (32 Million) ops/thread
+
+# Define indexes for each field in above row-format
+FLD_NUM_CLIENTS_IDX         = 0
+FLD_RUN_TYPE_IDX            = 1
+FLD_NUM_THREADS_IDX         = 2
+FLD_NUM_OPS_IDX             = 3
+FLD_ELAPSED_TIME_IDX        = 4
+FLD_AVG_ELAPSED_TIME_IDX    = 5
+FLD_SERVER_THROUGHPUT_IDX   = 6
+FLD_CLIENT_THROUGHPUT_IDX   = 7
+FLD_AVG_OPS_PER_THREAD_IDX  = 8
```

Comment:
New parsing interfaces to pluck metrics fields from comma-separated `Summary: ` line in stdout.